packages/lib/src/common/interfaces/options.ts

```diff
@@ -2,8 +2,8 @@ import { LayoutParams } from './layoutParams';
 import { BehaviourParams } from './behaviourParams';
 import { StylingParams } from './stylingParams';
 
-export interface StyleParams {
-  layoutParams: LayoutParams;
+export interface Options {
+  layoutParams?: LayoutParams;
```

Comment:
@yhattav I don't understand why I got a compilation error **before** I added this as mandaroty. Since the entire `options` in `GalleryProps` is **NOT** mandatory, I should not have got this error. 
Letting you know because I think we discussed that `layoutParams` is mandatory which is a good thing.